### DISCUSSION TITLE
[221680] Rocksmith® 2014 Edition - Remastered {仅本体 绝版游戏}

### DISCUSSION THREAD

**Discussion by kkaka-arch** [pjy612/SteamManifestCache#794] [Share]
### Steam 游戏ID

221680

### 密钥列表

```hash
221681;2774295430175ffb239a4064516349a5bde16e081b50bef3b5c8a3f0995acb34
```

### 清单附件

[221681_8947385950072815193.zip](https://github.com/user-attachments/files/16897284/221681_8947385950072815193.zip)

### 备注

这个是用电吉他玩的音乐游戏 因为游戏中音乐版权到期所以在steam上下架了 是绝版游戏 我还有几百个DLC 纯手动上传太累了 不过不影响游玩

**Comment by ZEERDEER** [comment]
👍
